packages/components/src/templates/dialog-samples.stories.ts

```diff
@@ -0,0 +1,129 @@
+                  Detailed information and sources of supply for Union Investment funds
+                </h4>
+                <p class="sd-paragraph">
+                  For detailed product-specific information and information on the opportunities and risks of the Union
+                  Investment funds mentioned, please refer to the current sales prospectuses, the investment conditions,
+                  the basic information sheets and the annual and semi-annual reports, which you can download free of
+                  charge in German Union Investment Institutional GmbH, Weißfrauenstraße 7, 60311 Frankfurt am Main,
+                  telephone +49 69 2567-7652, fax +49 69 2567-1010, institutional@union-investment.de,
+                  https://institutional.union-investment.de, receive. These documents form the only binding basis for
+                  the purchase. You can find a summary of your investor rights in German and further information on
+                  collective redress instruments under “Notices and Complaints”. Union Investment Institutional GmbH may
+                  decide at any time to revise any arrangements it may have made for the distribution of shares of a
+                  fund and/or share classes of a fund in a Member State other than its home Member State, under the
+                  conditions of Article 93a of Directive 2009/65/ EC and Article 32a of Directive 2011/61/EU.
+                </p>
+                <p class="sd-paragraph">
+                  The status of all information, representations and explanations are stated on the documents.
+                </p>
+
+                <div class="flex justify-end">
+                  <sd-button variant="secondary" size="lg" type="button">
+                    Download PDF
+                    <sd-icon library="global-resources" name="system/download" slot="icon-left"></sd-icon>
+                  </sd-button>
+                </div>
+              </div>
+            </sd-accordion>
+            <sd-accordion summary="Terms of Use VAG Reporting">
+              <div class="slot slot--border slot--text h-16">Default slot</div>
+            </sd-accordion>
+            <sd-accordion summary="Terms of use CRR and sustainability reporting">
+              <div class="slot slot--border slot--text h-16">Default slot</div>
+            </sd-accordion>
+          </sd-accordion-group>
+        </div>
+
+        <div slot="footer" class="flex flex-col w-full space-y-5">
+          <sd-button variant="primary" size="lg" type="button"> Accept terms of use </sd-button>
+          <sd-button variant="secondary" size="lg" type="button"> Download terms of use as PDF </sd-button>
+          <sd-button variant="secondary" size="lg" type="button"> Exit IK portal </sd-button>
+          <p class="sd-paragraph text-center">
+            If you have any questions or concerns, <sd-link href="#">contact us</sd-link>.
```

Comment:
text color black should be deemphasized. pls use text.neutral-700 instead of text.black